fiware-region-sanity-tests/commons/http_phonehome_server.py

```diff
@@ -113,8 +115,19 @@ def before_request_body():
     """
     Add a Tool to our new Toolbox.
     """
-    transaction_id = "txid:" + cherrypy.request.headers['TransactionId']
+    logger.info("before_request_body: %s ", cherrypy.request.params)
+    if 'TransactionId' in cherrypy.request.headers:
+        transaction_id = cherrypy.request.headers['TransactionId']
+    elif 'TransactionId' in cherrypy.request.params:
+        transaction_id = cherrypy.request.params['TransactionId']
+        cherrypy.request.params = {}
```

Comment:
Please, could you add 'TransactionId' as a constant? For example, named 'txIdHeader'.